extmod/uzlib/uzlib.h

```diff
@@ -0,0 +1,145 @@
+/*
+ * uzlib  -  tiny deflate/inflate library (deflate, gzip, zlib)
+ *
+ * Copyright (c) 2003 by Joergen Ibsen / Jibz
+ * All Rights Reserved
+ * http://www.ibsensoftware.com/
+ *
+ * Copyright (c) 2014-2018 by Paul Sokolovsky
+ */
```

Comment:
Does it make sense to add a license text here, so there is no doubt as to the licensing?

Reply:
> Does it make sense to add a license text here, so there is no doubt as to the licensing?

Making corresponding changes, to release 2.9.1.